service_add.py

```diff
@@ -71,11 +72,6 @@ def main():
         content = content.replace("<CONFIGDIR>", config_dir)
         content = content.replace("<STOPARGS>", " --config_dir " + str(config_dir))
 
-        print("Content is :")
```

Comment:
Any specific reason why you have removed this here? It would be part of the log of the service